examples/erc20/DAI/DAI.xml

```diff
@@ -167,9 +188,9 @@ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
             </ts:view>
         </ts:card>
 
-        <ts:card type="activity" name="aETHRepaidFromAnotherUser">
+        <ts:card type="activity" name="daiRepaidFromAnotherUser">
             <ts:origins>
-                <ethereum:event type="Repay" filter="&amp;(ownerAddress=${_user}(!(ownerAddress=${_repayer})))"/>
+                <ethereum:event type="Repay" filter="&amp;(ownerAddress=${_user}(!(ownerAddress=${_repayer})))" contract="aDAI"/>
```

Comment:
This filter isn't supported unfortunately. Also might not be possible with fetching event logs (need to check after the train).

Reply:
Dropped for demo purposes